src/axom/primal/tests/primal_clip.cpp

```diff
@@ -429,7 +429,7 @@ TEST(primal_clip, clip_oct_tet_omp)
   #endif /* AXOM_USE_OPENMP */
 
   #if defined(AXOM_USE_CUDA)
-AXOM_CUDA_TEST(primal_clip, unit_poly_clip_vertices_gpu)
+AXOM_CUDA_TEST(primal_clip, unit_poly_clip_vertices_cuda)
```

Comment:
Maybe not? I'm not sure. What does the `AXOM_CUDA_TEST` macro do? I used the `TEST()` macro for HIP, and didn't notice any issues.

https://github.com/LLNL/axom/blob/d8b76ec8bca08803563f966d6c0fc3691593f7c2/src/axom/core/Macros.hpp#L90-L102

Reply:
I don't think we really need GPU-specific test macros. If we replace them with standard GTEST macros, this should be done in a separate PR. We used to have something similar in RAJA, but have ripped them out in recent test restructuring.

Reply:
I think this macro works around an issue where NVCC requires that host-device lambdas are defined within a publicly-accessible function (either public member function or global function), since the gtest macros wrap the test body as a private member function.